### PR TITLE
Fix: Desc sorting for deposit and withdrawals

### PR DESCRIPTION
**Pull request by xenirio**

### CODE REVIEW

File: app/api/v2/admin/deposits.rb [hunk 1]
```diff
@@ -28,12 +28,18 @@ class Deposits < Grape::API
           optional :blockchain_key,
                    values: { value: -> { ::Blockchain.pluck(:key) }, message: 'admin.beneficiary.blockchain_key_doesnt_exist' },
                    desc: 'Blockchain key of the requested deposit'
+          optional :ordering,
+                   values: { value: %w(asc desc), message: 'admin.pagination.invalid_ordering' },
+                   default: 'desc',
```
Comment (dpatsora) [inline app/api/v2/admin/deposits.rb:33]: I would default to 'acs'. Just prepare the patch on FE

File: app/api/v2/admin/deposits.rb [hunk 2]
```diff
@@ -28,12 +28,18 @@ class Deposits < Grape::API
...
+                   desc: 'If set, returned values will be sorted in specific order, defaults to \'desc\'.'
+          optional :order_by,
+                   default: 'created_at',
+                   desc: 'Name of the field, which result will be ordered by.'
           use :uid
           use :currency
           use :currency_type
           use :date_picker
           use :pagination
-          use :ordering
```
Comment (dpatsora) [inline app/api/v2/admin/deposits.rb:36]: Why did you decide to remove it?

File: app/api/v2/admin/deposits.rb [hunk 3]
```diff
@@ -28,12 +28,18 @@ class Deposits < Grape::API
           optional :blockchain_key,
                    values: { value: -> { ::Blockchain.pluck(:key) }, message: 'admin.beneficiary.blockchain_key_doesnt_exist' },
                    desc: 'Blockchain key of the requested deposit'
+          optional :ordering,
+                   values: { value: %w(asc desc), message: 'admin.pagination.invalid_ordering' },
+                   default: 'desc',
+                   desc: 'If set, returned values will be sorted in specific order, defaults to \'desc\'.'
+          optional :order_by,
+                   default: 'created_at',
```
Comment (dpatsora) [inline app/api/v2/admin/deposits.rb:36]: I would default to 'id'

File: app/api/v2/management/deposits.rb [hunk 4]
```diff
@@ -18,12 +18,23 @@ class Deposits < Grape::API
           optional :page, type: Integer, default: 1, integer_gt_zero: true, desc: 'The page number (defaults to 1).'
           optional :limit, type: Integer, default: 100, range: 1..1000, desc: 'The number of deposits per page (defaults to 100, maximum is 1000).'
           optional :state, type: String, values: -> { ::Deposit.aasm.states.map(&:name).map(&:to_s) }, desc: 'The state to filter by.'
+          optional :ordering,
+                   values: { value: %w[asc desc], message: 'management.pagination.invalid_ordering' },
+                   default: 'desc',
+                   desc: 'If set, returned values will be sorted in specific order, defaults to \'desc\'.'
+          optional :order_by,
+                   default: 'created_at',
+                   desc: 'Name of the field, which result will be ordered by.'
```
Comment (dpatsora) [inline app/api/v2/management/deposits.rb]: Here, better you default to ordering 'desc' by 'id' to keep the old logic

File: app/api/v2/admin/deposits.rb (hunk 1 above)
Comment (calj) [reply app/api/v2/admin/deposits.rb:33]: desc is usually better so you get the latest first

File: app/api/v2/admin/deposits.rb (hunk 2 above)
Comment (xenirio) [reply app/api/v2/admin/deposits.rb:36]: The original of :ordering is default to `ordering: desc` and `order_by: id`. So, I need to change for both default.

Comment (dpatsora) [reply app/api/v2/admin/deposits.rb:36]: agree

**Comment by mod** [comment]
@xenirio you don't need to fork to push code in branches here

**Pull request closed without merging**